packages/governance/src/contractGovernance/governFilter.js

```diff
@@ -0,0 +1,117 @@
+// @ts-check
+
+import { E } from '@endo/eventual-send';
+import { Far } from '@endo/marshal';
+import { fit, keyEQ } from '@agoric/store';
+import { M } from '@agoric/vat-data';
+
+import {
+  ChoiceMethod,
+  QuorumRule,
+  ElectionType,
+  coerceQuestionSpec,
+} from '../question.js';
+
+const { details: X, quote: q } = assert;
+
+/**
+ * Make a pair of positions for a question about whether to update the offer
+ * filter. If the vote passes, the list of blocked invitation strings will be
+ * updated.
+ *
+ * @param {string[]} strings
+ */
+const makeOfferFilterPositions = strings => {
+  const positive = harden({ strings });
+  const negative = harden({ dontUpdate: strings });
```

Comment:
> My view is that the negative position should say what it's voting no on. When there are three options on the ballot, each should be self-describing. They shouldn't be meaningful only by reference to the other positions.

My view is that the position should say what the result of voting for it will be. The positive says "these will now be the filters". The negative can only say "the filters will not change". That "dontUpdate" has a list of strings can easily be read as some qualification on what not to update.

You're the lead on governance so if my argument is not convincing then I defer to your view. Whatever principle you use to decide, that is a de facto design principle of governance. I ask that it be documented because it's part of the human API of positions. (Apologies if it is and I have missed it.)

Reply:
I added more details in the governance README.